src/test/test_proj_data_info.cxx

```diff
@@ -111,6 +111,9 @@ test_generic_proj_data_info(ProjDataInfo& proj_data_info)
 #ifdef STIR_OPENMP
 #pragma omp parallel for schedule(dynamic)
 #endif
+  float view_offset=proj_data_info.get_scanner_ptr()->get_default_intrinsic_tilt();
```

Comment:
are you really sure that you need to disable this test? It makes me feel very uncomfortable. I think with your current changes in the other files, it should work... If it doesn't, we probably need to fix something.

By the way, you'd have to keep the `pragma omp` just above the `for` loop of course.

Reply:
I am also a bit puzzled about this. In the test for a given bin.view_num, get_LOR calculates phi which should be phi+view_offset. Then the second part of the test uses get_bin to return the value of bin.view_offset from that phi which should give the actual value. But somewhere it seems this breaks as test_proj_data_info gives the following error:
 
 Error. round-trip get_LOR then get_bin (LORAs2Points): segment
	Problem at    segment = 15, axial pos 0, view = 189, tangential_pos_num = 30
	round-trip to segment = -15, axial pos 0, view = -3, tangential_pos_num = -30
Max Deviation:  segment = 30, axial pos 0, view = 192, tangential_pos_num = 62

I am unsure why this segment swap occurs with the current code.